Index: src/pt2_config.c
===================================================================
--- src/pt2_config.c	(revision 188)
+++ src/pt2_config.c	(working copy)
@@ -76,8 +76,10 @@
 	if (f != NULL)
 		proTrackerDotIniFound = true;
 
+	char *user_data = getenv("SNAP_USER_DATA");
+	
 	// check in ~/.protracker/
-	if (!proTrackerDotIniFound && changePathToHome() && chdir(".protracker") == 0)
+	if (!proTrackerDotIniFound && changePathToHome() && user_data && chdir(user_data) == 0)
 	{
 		f = fopen("protracker.ini", "r");
 		if (f != NULL)
@@ -106,7 +108,7 @@
 		ptDotConfigFound = true;
 
 	// check in ~/.protracker/
-	if (!ptDotConfigFound && changePathToHome() && chdir(".protracker") == 0)
+	if (!ptDotConfigFound && changePathToHome() && user_data && chdir(user_data) == 0)
 	{
 		f = openPTDotConfig();
 		if (f != NULL)
@@ -130,7 +132,7 @@
 	colorsDotIniFound = loadColorsDotIni();
 
 	// check in ~/.protracker/
-	if (!colorsDotIniFound && changePathToHome() && chdir(".protracker") == 0)
+	if (!colorsDotIniFound && changePathToHome() && user_data && chdir(user_data) == 0)
 		loadColorsDotIni();
 #endif
 
